pkg/fixtures/expecter.go

```diff
@@ -6,4 +6,5 @@ type Expecter interface {
 	ManyArgsReturns(str string, i int) (strs []string, err error)
 	Variadic(ints ...int) error
 	VariadicMany(i int, a string, intfs ...interface{}) error
+	VariadicNoReturn(j int, is ...interface{})
```

Comment:
I recommend creating a new interface fixture and then in `.mockery.yaml` add a new configuration section using the previously buggy config. This is going to assert that mockery generates valid code. Something like

```yaml
      VariadicNoReturnInterface:
        config:
          with-expecter: True
          unroll-variadic: False
```

This way the integration tests will run and assert the validity of the generated mocks.